### PR TITLE
Updated README with cute story

### PR DESCRIPTION
**Pull request by ghost**
DO YOU KNOW WHY AM I DOING THIS? Because I like it... Put in the main!
Hey, Naga, by the way... this is useful for us! PUT IT ON!

### CODE REVIEW

**Comment by cardozoaldama** [comment]
I loving how those users want to make a pull request! However, I'll see you code what you have.

File: README.md
Comment (cardozoaldama) [inline README.md:1]: Look at this! I can comment you in the hole page! That is **amazing**!

**Review by cardozoaldama — APPROVED** [review]
I think you cannot see those comments in my review, and that is something... weird...

File: README.md
Comment (ghost) [reply README.md:1]: I didn't see it! 🔢

**Comment by cardozoaldama** [comment]
I'll approve this, because I like the README.
Now, I recommend you to try mess something so I can **reject** your pull request.

**Pull request merged**